docroot/modules/custom/hs_paragraph_types/modules/hs_hero_image/config/install/core.entity_view_display.paragraph.hs_hero_image.default.yml

```diff
@@ -0,0 +1,24 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.field.paragraph.hs_hero_image.field_hs_hero_image
+    - paragraphs.paragraphs_type.hs_hero_image
+  module:
+    - stanford_media
+id: paragraph.hs_hero_image.default
+targetEntityType: paragraph
+bundle: hs_hero_image
+mode: default
+content:
+  field_hs_hero_image:
+    type: media_responsive_image_formatter
+    weight: 0
+    label: hidden
+    settings:
+      view_mode: default
+      image_style: hero_image
```

Comment:
This hero image is a part of the installation profile. If this module is to be encapsulated the image style will have to be included.

Reply:
Thanks. I knew i missed something.